src/UsgsAstroLsSensorModel.cpp

```diff
@@ -870,22 +873,27 @@ double UsgsAstroLsSensorModel::getImageTime(
 {
    // Flip image taken backwards
    double line1 = image_pt.line;
-   if (_data.m_ImageFlipFlag == 1)
-      line1 = _data.m_TotalLines - 1 - image_pt.line;

```

Comment:
As stated in the PR description, I removed the functional part of the image flip flag, but left the computation code in. Depeneding on how we want to handle removing that it can be added to this pr.